pkg/cmd/pipelineresource/describe.go

```diff
@@ -0,0 +1,110 @@
+// Copyright © 2019 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pipelineresource
+
+import (
+	"fmt"
+	"text/tabwriter"
+	"text/template"
+
+	"github.com/spf13/cobra"
+	"github.com/tektoncd/cli/pkg/cli"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	cliopts "k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+const templ = `Name:	{{ .PipelineResource.Name }}
+Namespace:	{{ .PipelineResource.Namespace }}
+PipelineResource Type:	{{ .PipelineResource.Spec.Type }}
+
+Params
+{{- $l := len .PipelineResource.Spec.Params }}{{ if eq $l 0 }}
+No params
+{{- else }}
+NAME	VALUE
+{{- range $i, $p := .PipelineResource.Spec.Params }}
+{{ $p.Name }}	{{ $p.Value }}
+{{- end }}
+{{- end }}
+
+Secret Params
+{{- $l := len .PipelineResource.Spec.SecretParams }}{{ if eq $l 0 }}
+No secret params
+{{- else }}
+FIELDNAME	SECRETNAME	SECRETKEY
+{{- range $i, $p := .PipelineResource.Spec.SecretParams }}
+{{ $p.FieldName }}	{{ $p.SecretName }}	{{ $p.SecretKey }}
```

Comment:
It's better to remove the `secret key` ?
The reason is, it may contain sensitive information. It should not be that easy to view such details.

Reply:
Addressed